src/main/scala/com/typesafe/sbt/osgi/Osgi.scala

```diff
@@ -16,19 +16,20 @@
 
 package com.typesafe.sbt.osgi
 
-import java.nio.file.{FileVisitOption, Files, Path}
+import java.nio.file.{ FileVisitOption, Files, Path }
```

Comment:
Imports were messed up in this PR, this fixes it https://github.com/sbt/sbt-osgi/pull/100/files#diff-2530a9c505dd4cb34217eb195d5db3cab065049f033db306f485078da72edd5dR19-R32